src/components/compassElevationSlider.tsx

```diff
@@ -0,0 +1,164 @@
+import React, { useEffect, useRef, useState } from "react";
+import { describeSlice } from "../utils/utils";
+
+type Props = {
+  elevation: number;
+  changeElevation: (value: number) => void;
+  radius: number;
+};
+
+const CompassElevationSlider: React.FC<Props> = ({
+  elevation,
+  changeElevation,
+  radius,
+}) => {
+  const visualMin = -30;
```

Comment:
put these consts in CAPS and outside of component. you dont need to define these vars every time the component rerenders